### PR TITLE
11.0 migrated base_partner_sequence

### PR DESCRIPTION
**Pull request by jesusVMayor**

### CODE REVIEW

File: base_partner_sequence/models/partner.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            default['ref'] = self._get_next_ref()
+        return super(ResPartner, self).copy(default)
+
+    @api.multi
+    def write(self, vals):
+        for partner in self:
+            if not vals.get('ref') and partner._needsRef(vals) and \
+               not partner.ref:
+                vals['ref'] = self._get_next_ref(vals=vals)
+
```
Comment (simoneversienti) [inline base_partner_sequence/models/partner.py]: Why is there this empty line?

File: base_partner_sequence/models/partner.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                vals['ref'] = self._get_next_ref(vals=vals)
+            super(ResPartner, partner).write(vals)
+        return True
+
+    @api.multi
+    def _needsRef(self, vals=None):
+        """
+        Checks whether a sequence value should be assigned to a partner's 'ref'
+
+        :param cr: database cursor
```
Comment (FrancoTampieri) [inline base_partner_sequence/models/partner.py]: Hi, remove from the comments the old reference to cr, uid, id if not used in the code

File: base_partner_sequence/models/partner.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        :param uid: current user id
```
Comment (FrancoTampieri) [inline base_partner_sequence/models/partner.py]: Hi, remove from the comments the old reference to cr, uid, id if not used in the code

File: base_partner_sequence/models/partner.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        :param id: id of the partner object
```
Comment (FrancoTampieri) [inline base_partner_sequence/models/partner.py]: Hi, remove from the comments the old reference to cr, uid, id if not used in the code

**Review by FrancoTampieri — CHANGES_REQUESTED** [review]

**Comment by FrancoTampieri** [comment]
@jesusVMayor I fixed the problem on the main repository, so you can rebase and apply the simply mods that I found in you PR.

Regards

Franco

**Comment by jesusVMayor** [comment]
fixed

**Review by yajo — APPROVED** [review]
Code review

**Comment by ddufresne** [comment]
:+1: code + functionnal

**Pull request merged**